src/container_specs/mod.rs

```diff
@@ -17,3 +19,43 @@ impl Default for SpecificationType {
         SpecificationType::Oci
     }
 }
+
+pub fn optional_ordered_map<S>(
+    value: &Option<HashMap<String, String>>,
+    serializer: S,
+) -> Result<S::Ok, S::Error>
+where
+    S: Serializer,
+{
+    use serde::Serialize;
```

Comment:
OOC: why does this `use` need to be inside the fn vs a top level import?

Reply:
I think we can move it to top-level. I had a compiler error that I thought moving this here fixed it, but apparently I can move it to top and still compiles.

Reply:
Awesome, lets do that 😄